src/GeoContacts/GeoContacts.iOS/Info.plist

```diff
@@ -46,11 +46,12 @@
 			<key>CFBundleTypeRole</key>
 			<string>Editor</string>
 			<key>CFBundleURLName</key>
-			<string>com.microsoft.advocates</string>
+			<string>com.microsoft.cloudadvocates</string>
```

Comment:
I would assume on the server we need to update the scripts to swap these out. Can you update the script in there.